spec/suites/map/handler/Map.KeyboardSpec.js

```diff
@@ -16,6 +16,9 @@ describe("Map.Keyboard", function () {
 			zoomAnimation: false	// If true, the test has to wait extra 250msec
 		});
 
+		// make keyboard-caused panning instant to cut down on test running time
+		map.panBy = function (offset) { return L.Map.prototype.panBy.call(this, offset, {animate: false}); };
+
```

Comment:
Does it make sense to remove everywhere `animate` only to have faster tests? The most users are using everything with animation, so tests should handle that too

Reply:
Not everywhere, but in places where the tests don't care whether it's animated or not. It's keyboard interactions that are being tested in this place, not `panBy` animation (which is tested elsewhere).